packages/flutter/test/material/date_range_picker_test.dart

```diff
@@ -268,6 +268,15 @@ void main() {
     });
   });
 
+  testWidgets('Long helpText cant not cutoffs the save button', (WidgetTester tester) async {
+    helpText = 'long helpText' * 100;
+    saveText = 'make it so';
+    await preparePicker(tester, (Future<DateTimeRange?> range) async {
+      expect(find.text(helpText!), findsOneWidget);
+      expect(find.text(saveText!), findsOneWidget);
```

Comment:
It looks like this test will fail on master due to layout overflow, not due to these expectations. So I would add this line here: `expect(tester.takeException(), null);`

Reply:
@justinmc Thanks for your review, I have updated the test.